sechub-scan/src/test/java/com/daimler/sechub/domain/scan/HTMLScanResultReportModelBuilderTest.java

```diff
@@ -23,145 +25,181 @@
+
+        when(scanResult.getInfo()).thenReturn("theInfo");
+        when(scanResult.getJobUUID()).thenReturn(uuid);
+        when(scanResult.getTrafficLight()).thenReturn("trafficlight1");
+        builderToTest.webDesignMode = true;
+        builderToTest.embeddedCSS = "embeddedCssContent";
+        Resource cssResource = mock(Resource.class);
+        File expectedFile = ScanDomainTestFileSupport.getTestfileSupport()
+                .createFileFromRoot("sechub-scan/src/main/resources/templates/report/html/scanresult.css");
+        when(cssResource.getFile()).thenReturn(expectedFile);
+        builderToTest.cssResource = cssResource;
+
+        /* execute */
+        Map<String, Object> map = builderToTest.build(scanResult);
+
+        /* test */
+        assertEquals("theInfo", map.get("info"));
+        assertSame(result, map.get("result"));
+        assertSame(greenList, map.get("greenList"));
+        assertSame(redList, map.get("redList"));
+        assertSame(yellowList, map.get("yellowList"));
+        assertEquals(true, map.get("isWebDesignMode"));
+
+        // check css ref for webdesign mode
+        assertNotNull(map.get("includedCSSRef"));
+        String path = (String) map.get("includedCSSRef");
+        File foundFile = new File(path);
+
+        assertEquals(expectedFile.getCanonicalPath(), foundFile.getCanonicalPath());
+
+        assertEquals("trafficlight1", map.get("trafficlight"));
+        assertEquals(uuid.toString(), map.get("jobuuid"));
+        assertEquals(HIDE_LIGHT, map.get("styleRed"));
+        assertEquals(HIDE_LIGHT, map.get("styleYellow"));
+        assertEquals(HIDE_LIGHT, map.get("styleGreen"));
+    }
+
+    @Test
+    public void trafficlight_red_set_display_block__others_are_none() {
+        when(scanResult.getTrafficLight()).thenReturn(TrafficLight.RED.name());
+
+        Map<String, Object> map = builderToTest.build(scanResult);
+        assertEquals(SHOW_LIGHT, map.get("styleRed"));
+        assertEquals(HIDE_LIGHT, map.get("styleYellow"));
+        assertEquals(HIDE_LIGHT, map.get("styleGreen"));
+    }
+
+    @Test
+    public void trafficlight_yellow_set_display_block__others_are_none() {
+        when(scanResult.getTrafficLight()).thenReturn(TrafficLight.YELLOW.name());
+
+        Map<String, Object> map = builderToTest.build(scanResult);
+        assertEquals(HIDE_LIGHT, map.get("styleRed"));
+        assertEquals(SHOW_LIGHT, map.get("styleYellow"));
+        assertEquals(HIDE_LIGHT, map.get("styleGreen"));
+    }
+
+    @Test
+    public void trafficlight_green_set_display_block__others_are_none() {
+        when(scanResult.getTrafficLight()).thenReturn(TrafficLight.GREEN.name());
+
+        Map<String, Object> map = builderToTest.build(scanResult);
+        assertEquals(HIDE_LIGHT, map.get("styleRed"));
+        assertEquals(HIDE_LIGHT, map.get("styleYellow"));
+        assertEquals(SHOW_LIGHT, map.get("styleGreen"));
+    }
+
+    @Test
+    public void test_code_scan_entries() {
+
```

Comment:
Could you please add

/* prepare */

/* execute */

/* test */

And "test_code_scan_entries" starts with "test" which is a little bit redundant inside tests (junit3 needed this, but junit4 is free in method names)

"test_code_scan_entries" - sorry. but I don't exactly get - while just giving a glance - what the test is really meant to do here, what's the purpose. Maybe a dedicated test method name would make it more clear?

Reply:
done